lib/rubygems/commands/cert_command.rb

```diff
@@ -290,6 +303,13 @@ def sign_certificates # :nodoc:
     end
   end
 
+  def re_sign_cert(cert, private_key)
+    Gem::Security::Signer.re_sign_cert(cert, private_key) do |cert_path, expired_cert_path|
```

Comment:
Since this needs the certificate location to store the expired certificate when we generate a new one that should be passed in as well, then have Gem::Security::Signer::re_sign_cert accept three arguments.

Reply:
ok